Package.swift

```diff
@@ -57,7 +57,8 @@ let package = Package(
                 .product(name: "NIOTransportServices", package: "swift-nio-transport-services"),
                 .product(name: "NIOFoundationCompat", package: "swift-nio"),
                 .product(name: "JMESPath", package: "jmespath.swift"),
-            ]
+            ],
+            swiftSettings: [.unsafeFlags(["-Xfrontend", "-warn-concurrency"])]
```

Comment:
Is this to make it easy to turn strict on? I think warn is the default right?

Reply:
How are you turning strict on. This is how I thought it was done.